client/src/components/Header/Header.jsx

```diff
@@ -49,7 +68,7 @@ export default function Header({ darkMode, handleThemeChange }) {
 
   const buttonList = [
     { text: 'Utwórz nowe konto', icon: <HowToRegIcon />, onClick: handleSignUp },
-    { text: 'Wyloguj się', icon: <LogoutIcon />, onClick: handleLogOut }
```

Comment:
I would not delete it, after all this the only button that should stay in the menu